Branch-TestBed/Branch-TestBedUITests/Branch_TestBedUITests.m

```diff
@@ -0,0 +1,74 @@
+//
+//  Branch_TestBedUITests.m
+//  Branch-TestBedUITests
+//
+//  Created by Parth Kalavadia on 8/2/17.
+//  Copyright © 2017 Branch Metrics. All rights reserved.
+//
+#import <XCTest/XCTest.h>
+#define DEEPLINK_SLEEP 10
```

Comment:
When defining constants, it's best to *not* use defines and to use typed const variables so that type checking and expected use are explicit. So this variable would be best declared as 

`static NSTimeInterval const kDeepLinkSleepTimeInterval = 10.0;`.

The `static` part means that its scope is limited to the file, and the `const` part means the value can't be changed.  Also, don't upper case variable names unless someone makes you.  You're not writing COBOL code.

You can preface constants with a package prefix or `k` if the variable is local to the file if you like.  The `k` is from the German word "Konstante", which is used in math texts and somehow got carried over to C programming.